frontend/test/validate-e2e-test-files.js

```diff
@@ -0,0 +1,20 @@
+const path = require("path");
```

Comment:
Do we want to add Node shabang here?
```suggestion
#!/usr/bin/env node

const path = require("path");
```

Reply:
Oh right, thank you!